packages/react/src/section-alert/docs/overview.mdx

```diff
@@ -39,234 +40,180 @@ Section alerts appear inside of the main content area of the page and are used t
+Use warning section alerts (orange) to tell the user something urgent. Only use an alert if the information will help the user avoid a problem.
 
 ```jsx live
 <Stack gap={1}>
 	<SectionAlert title="A warning message for this section" tone="warningHigh" />
-	<SectionAlert
-		title="A warning message for this section"
-		tone="warningMedium"
-	/>
-	<SectionAlert title="A warning message for this section" tone="warningLow" />
 </Stack>
 ```
 
-### Info
+### Information
 
-The information section alert can be used to alert the user on neutral messages.
+Use info section alerts (blue) to highlight information which is important for users.
 
 ```jsx live
 <Stack gap={1}>
-	<SectionAlert title="Please read the comments carefully" tone="infoHigh" />
-	<SectionAlert title="Please read the comments carefully" tone="infoMedium" />
-	<SectionAlert title="Please read the comments carefully" tone="infoLow" />
+	<SectionAlert title="These records need more information" tone="infoHigh" />
 </Stack>
 ```
 
-## Dismissable
-
-Section alerts can be dismissed by a user if they have understood the message and no longer need to see it. Only use the close action when the alert does not impact the user’s available actions.
-Avoid dismissing error messages which block the user from completing a task. Instead, provide a way for the user to fix the error, then dismiss the alert once the error has been resolved.
-
-Use the `onClose` prop to make the alert dismissable.
+### In progress
 
-When a user dismisses an alert, it’s important to manage their focus appropriately to help them continue their task or journey. In some cases, there may not be an obvious (or any) element to send focus to, such as when the alert was triggered by an action on a previous page or from a drawer that is now closed. Always carefully consider the context and previous user actions before moving their focus to ensure a smooth and accessible experience.
+Use in progress section alerts to highlight information which is relates to status.
 
 ```jsx live
-<SectionAlert
-	onClose={console.log}
-	title="Section alert title"
-	tone="successHigh"
-/>
+<Stack gap={1}>
+	<SectionAlert title="Record is in progress" tone="inProgressLow" />
+</Stack>
 ```
 
-## Composition
+## Status and emphasis
 
-Section alerts are composed of an icon, title, description, action and close action.
+The tone prop combines both a status and emphasis, which sets the correct importance and severity of the Section alert by automatically applying appropriate colour, iconography and accessible descriptions.
 
-The icon and title are mandatory and achieve a compact alert, but we encourage the use of description when additional information might be helpful to the user.
+There are nine unique tones with five levels of emphasis. All tones are available in Low emphasis but others are only available in Medium and High.
```

Comment:
I notice the same paragraph in Status Badge but that one has a typo "a" instead of "are"